### PR TITLE
fix #184: make status text more clear

### LINKED ISSUE #184: Make work in progress text more clear
Currently when OMEMO support is still in progress it shows a bookmark icon. The hover text says "Client is working on OMEMO support". First, client itself can't work on something. This is done by people, developers, so semantically this is not right, in my opinion. This is even more confusing to not native English speakers (as i understood from a recent chat on Spark support forum). They read this like "Client is working with OMEMO support". They don't get the nuance of "on". They see "Client is working" and the quickly conclude, that it should work. So, i would suggest changing that to something like "Developers working to support OMEMO". Yeah, this still can be confusing, but maybe a bit more clear and correct. Maybe something like "No support for OMEMO yet" would be even more clear. Maybe also changing the icon to some tools or construction sign ("under construction") also makes sense.

### PR DESCRIPTION
**Pull request by wrooot**

### CODE REVIEW

**Comment by Echolon** [comment]
@wrooot thank you very much!

As you see, I need to wait on @bascht to repair this issue first!

**Comment by bascht** [comment]
@wrooot @Echolon sorry for the holdup. Just a tiny thing: Can we reword it to »Developer(s) are working« ?

**Comment by wrooot** [comment]
Sure. Changed.

**Comment by bascht** [comment]
@wrooot Great, thank you! :tulip:
